pdf-aam/index.html

```diff
@@ -226,81 +183,549 @@ <h4 id="role-map-example"><code>example</code></h4>
                   <abbr title="macOS Accessibility Protocol">AX API</abbr><sup>[<a href="#ftn.note1">Note 1</a>]</sup>
                 </th>
                 <td>
-                  <span class="property">AXRole: <code>AXxample</code></span
+                  <span class="property">AXRole: <code>Use WAI-ARIA mapping</code></span
+                  ><br />
+                  <span class="property">AXSubrole: <code>Use WAI-ARIA mapping</code></span
+                  ><br />
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="Notes">Notes:</abbr></th>
+                <td>
+                  <span class="property"
+                    >Note from PDFa:
+                    <code
+                      >Added note - question for the group... A screen reader shouldn't "announce" when it hits the first Document tag but should it "announce" when it hits "nested" (subsequent)
+                      Document tags, to tell someone they are in a "second" document? AND... the same question applies to Document Fragment, too. If "Yes," how do we indicate that in the AAM?</code
+                    ></span
                   ><br />
-                  <span class="property">AXSubrole: <code></code></span><br />
                 </td>
               </tr>
             </tbody>
           </table>
 
-          <div class="note" id="ftn.note1">
-            <p>
-              <sup>[Note 1]</sup>
-              Footnote content
-            </p>
-          </div>
-        </section>
-      </section>
+          <h4 id="role-map-example"><code>Sect</code></h4>
+          <table class="data" aria-labelledby="role-map-example">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="role-reference" href="https://www.w3.org/TR/core-aam-1.2/#role-map-region   https://www.w3.org/TR/core-aam-1.2/#role-map-generic"
+                    ><code>region "region" IF there is an accessible name. Otherwise use generic</code></a
+                  >
+                </td>
+              </tr>
+              <tr>
+                <th>Analogous HTML Element</th>
+                <td>
+                  <span class="property" href="https://www.w3.org/TR/html-aam-1.0/#el-section ">Role: <code>section </code></span><br />
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property">Role: <code>Use WAI-ARIA mapping</code></span
```

Comment:
ditto for this sectino... you don't need to list "Use another mapping" at the API property level... (Role, Control Type, AXRole, etc.) Just add it as text:

```html
<div class="general">Use WAI-ARIA mapping</div>
```